### DISCUSSION TITLE
perguntas

### DISCUSSION THREAD

**Discussion by marcoscode404** [marcoscode404/igniteFeed#2] [Q&A] [answered]
Tenho uma dúvida de como criar uma nova porta de acesso no Vite! 
parte do código `

  import { defineConfig } from 'vite'
import react from '@vitejs/plugin-react'

// https://vitejs.dev/config/
export default defineConfig({
  plugins: [react()],
})

`

**Comment by Viniciusmg99** [comment, marked as answer]
Configura assim mano   
"scripts": {
    "start": "PORT=8000 react-scripts start",
    "build": "react-scripts build",
    "test": "react-scripts test",
    "eject": "react-scripts eject"
  },

**Comment by marcoscode404** [comment]
vlw deu certo
